### PR TITLE
create brew-services extension

### PR DESCRIPTION
**Pull request by es183923**
## Description

Adds an extension to manage brew services.
redo of #68
- [x] view brew services
- [x] start, stop, restart, and run a brew service
- [x] show in finder/copy plist file path
- [x] show when a brew service doesn't run successfully
- [x] automatically detects brew installation location (can choose a custom location in preferences)
- [ ] run services that require `sudo`
- [x] has a good icon
- [x] shows error when there are no services
- [x] shows error when `brew` is not installed
## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/72744903/137605006-c87de13e-46d8-4c71-a44d-11c75a63c9cb.mp4

## Checklist

- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/brew-services/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "name": "brew-services",
+  "title": "Manage Brew Services",
+  "description": "Manage homebrew services",
+  "icon": "command-icon.png",
+  "author": "es183923",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "manage-services",
+      "title": "Manage Brew Services",
```
Comment (mattisssa) [inline extensions/brew-services/package.json:11]:
```suggestion
      "title": "Manage Services",
```

**Comment by es183923** [comment]
Added the brew icon. Taken from the [brew repo](https://github.com/Homebrew/brew.sh/blob/master/assets/img/homebrew.svg)

**Comment by es183923** [comment]
I'd like for some more people to test this extension before it gets merged. I'm not sure that it will always work.

**Review by mattisssa — APPROVED** [review]

**Comment by es183923** [comment]
After doing some testing by myself, I think it is works fine. Ready to merge! @mattisssa

**Review by mattisssa — APPROVED** [review]

**Pull request merged**